lib/src/components/NxTransferList/TransferListHalf.scss

```diff
@@ -118,6 +127,43 @@ $-visible-item-count: 10;
   }
 }
 
+.nx-transfer-list__item--with-reordering {
+  .nx-transfer-list__select {
+    border-radius: var(--nx-border-radius);
```

Comment:
Isn't this redundant?

Reply:
> Isn't this redundant?

Yup! Removed: 8da7e29ba73263e6ce9f96e12d36ba9915305d0f